stories/container.stories.js

```diff
@@ -100,34 +85,150 @@ storiesOf('HURUmap UI|ChartContainer', module)
               />
             </ChartContainer>
           </Grid>
-          <EmbedPopup
-            anchorEl={shareAnchorEl}
-            onClose={handleCloseShare}
-            open={shareAnchorEl !== null}
-            title="Embed code for this chart"
-            subtitle="Copy the code below, then paste into your own CMS or HTML. Embedded charts are responsive to your page width, and have been tested in Firefox, Safari, Chrome, and Edge."
+        </Grid>
+      );
+    })
+  )
+  .add('Hidden Drop Downs', () =>
+    React.createElement(() => {
+      return (
+        <Grid
+          container
+          justify="center"
+          alignItems="center"
+          style={{ background: 'whitesmoke', height: '100%' }}
+        >
+          <Grid
+            item
+            xs={select(
+              'xs',
+              ['auto', 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12],
+              12
+            )}
+            md={select(
+              'md',
+              ['auto', 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12],
+              6
+            )}
           >
-            {`<iframe
-  id="cr-embed-region-11-literacy_and_numeracy_tests-english_test_dist"
-  className="census-reporter-embed"
-  src="https://tanzania.hurumap.org/embed/iframe.html?geoID=region-11&geoVersion=2009&chartDataID=literacy_and_numeracy_tests-english_test_dist&dataYear=2015&chartType=pie&chartHeight=200&chartQualifier=&chartRelease=Uwezo+Annual+Assessment+Report+2015&chartSourceTitle=&chartSourceLink=&chartTitle=Percentage+of+children+aged+6-16+passing+English+literacy+tests&chartSubtitle=&initialSort=-value&statType=percentage"
-  frameBorder="0"
-  width="100%"
-  height="300"
-  style="margin: 1em; max-width: 300px;"
-/>
-<script src="https://tanzania.hurumap.org/static/js/embed.chart.make.js" />`}
-          </EmbedPopup>
-          <InfoPopup
-            anchorEl={infoAnchorEl}
-            onClose={handleCloseInfo}
-            onExploreData={handleExploreData}
-            open={infoAnchorEl !== null}
-            sourceLink="https://codeforafrica.org"
-            sourceTitle="Code for Africa"
+            <ChartContainer
+              infoComponent={null}
+              content={{
+                height: 338
+              }}
+              loading={boolean('loading', true)}
+              shareComponent={null}
```

Comment:
1- Am thinking you should pass a click callback and then handle your own component pop up instead of giving us a `shareComponent` or `infoComponent`. 

2- Also we might need to have more buttons or less custom action buttons. See this issue by @DavidLemayian 

https://www.pivotaltracker.com/n/projects/2204613

sorry @kilemensi, its always with these additional functionalities.

Reply:
1. Less is handled by current design @karimkawambwa ... Just set the component you don't want to `null`.
2. To handle more, means we need to accept _an array_ of handlers (click callbacks) or buttons... I'm leaning towards buttons since Chart container wouldn't need to handle customization of those button e.g. icons, etc. They'd be supplied to the chart container already customized.